src/components/molecules/Layout/DefaultLayout.js

```diff
@@ -0,0 +1,24 @@
+import React, {Component, Fragment} from 'react';
```

Comment:
We could have this as a template. on the path `src/components/templates/NavBar` considering it does not house any data